### ISSUE TITLE
Windows Support

### ISSUE THREAD

**Issue by ControlCplusControlV** [FuelLabs/fuel-core#463]
Support Windows eventually, not a priority atm. Maybe add some CI Job so that compatibility is ensured. At the time of writing this is blocked by in graph_api.rs](https://github.com/FuelLabs/fuel-core/blob/master/fuel-core/src/service/graph_api.rs#L25). But this is very bottom of priority list so really whenever anyhow has free time
